src/main/java/teammates/common/datatransfer/attributes/InstructorAttributes.java

```diff
@@ -142,12 +144,12 @@ public String getIdentificationString() {
 
     @Override
     public String getEntityTypeAsString() {
-        return "Instructor";
+        return DEFAULT_DISPLAY_NAME;
```

Comment:
I don't think `DEFAULT_DISPLAY_NAME` exists in every attribute.

Reply:
Should I create a new variable, "ATTRIBUTE_NAME" for each attribute in their respective classes?

Reply:
Sounds good.